app/components/access_panels/library_location_component.html.erb

```diff
@@ -38,7 +38,7 @@
           <% end %>
         </td>
         <% if item.present? %>
-          <td class="item-availability" data-live-lookup-id="<%= document[:id] %>" data-status-target=".availability-icon" data-barcode="<%= item.barcode if item.live_status? %>" <%= "data-request-url='#{helpers.request_url(document, library: item.library, location: item.home_location, barcode: item.barcode)}'".html_safe if !location_request_link.render? && item.circulates? && !item.request_link.render? %>>
+          <td class="item-availability" data-live-lookup-id="<%= document.live_lookup_id %>" data-status-target=".availability-icon" data-item-id="<%= document.barcode_to_uuid_mapping.fetch(item.barcode, item.barcode) if item.live_status? %>" <%= "data-request-url='#{helpers.request_url(document, library: item.library, location: item.home_location, barcode: item.barcode)}'".html_safe if !location_request_link.render? && item.circulates? && !item.request_link.render? %>>
```

Comment:
Is `item` here a `Holdings::Item`? Maybe wecould give it a `live_lookup_item_id` accessor and bury the complexity in the model?

Reply:
Hmm...maybe, I'll take a look.